tests/cases/fourslash/referencesBloomFilters.ts

```diff
@@ -16,8 +16,8 @@
 
 const ranges = test.ranges();
 const [r0, r1, r2, r3] = ranges;
-verify.referenceGroups(r0, [{ definition: "(property) searchProp: number", ranges }]);
-verify.referenceGroups([r1, r2], [{ definition: "(property) searchProp: number", ranges }]);
+verify.referenceGroups(r0, [{ definition: "(property) searchProp: number", ranges: [r0, r1, r3, r2] }]);

```

Comment:
Out of curiosity, why do these reference groups ordering change?

Reply:
The "fake" file system previously used by fourslash uses the default insertion order behavior of Map, however the new vfs implementation uses a custom `SortedMap` that ensures entries are sorted by key. This aligns more closely with how most file systems return results via `fs.readdirSync()`.